examples/nodejs/latency.ts

```diff
@@ -0,0 +1,56 @@
+import { CharactrAPISDK } from "../../lib/sdk";
+import { credentials } from "../credentials.js";
+
+async function nodeLatencyExample() {
+  const sdk = new CharactrAPISDK(credentials);
+
+  await sdk.init();
+
+  console.log(`Running latency test. Today is ${new Date().toUTCString()}`);
+
+  // get server latency
+  const serverStart = Date.now();
+  await fetch("https://api.gemelo.ai/", { method: "HEAD" });
+  const serverLatency = Date.now() - serverStart;
+  console.log(`serverLatency is ${serverLatency} ms`);
+
+  // get voice latencies
+  const voices = [20, 40, 151, 222];
+  for (const v of voices) {
+    let startTime = 0;
+    let timeToFirstByte = 0;
+    const stream = await sdk.tts.convertStreamDuplex(v, {
+      onData: (data) => {
```

Comment:
`data` is an unused variable. Please, run `yarn lint` and correct warnings/errors it prints out if possible.